src/main/webapp/WEB-INF/view/product/all-products.jsp

```diff
@@ -26,12 +26,14 @@
             </td>
             <td>
                 <a href="${pageContext.request.contextPath}/cart/products/add?id=${product.id}">To cart</a>
-                <a href="${pageContext.request.contextPath}/products/delete?id=${product.id}">Delete</a>
             </td>
         </tr>
     </c:forEach>
 </table>
 <a href="${pageContext.request.contextPath}/products/add">Add product</a>
+<a href="${pageContext.request.contextPath}/orders">My orders</a>
+<a href="${pageContext.request.contextPath}/admin/orders">Admin orders</a>
+<a href="${pageContext.request.contextPath}/admin/products">Admin products</a>
```

Comment:
let's remove this links from here
this page will be available to user - we don't want him to go on these pages

Reply:
done